graylog2-server/src/main/java/org/graylog2/bindings/providers/MongoConnectionProvider.java

```diff
@@ -19,19 +19,27 @@
 import org.graylog2.configuration.MongoDbConfiguration;
 import org.graylog2.database.MongoConnection;
 import org.graylog2.database.MongoConnectionImpl;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import javax.inject.Inject;
 import javax.inject.Provider;
 
 public class MongoConnectionProvider implements Provider<MongoConnection> {
+    private static final Logger LOG = LoggerFactory.getLogger(MongoConnectionProvider.class);
     private static MongoConnection mongoConnection = null;
```

Comment:
I wonder why I made that field `static` in the first place. It should've be `private final` instead.